### PR TITLE
Nette 3.0 compatibility

### PR DESCRIPTION
**Pull request by andrejsoucek**
- Updated dependencies
- Updated code to match Nette 3.0
- Updated tests
Check the onMatch deletion, please.

### CODE REVIEW

**Comment by paveljanda** [comment]
Thank you, I will check that

**Comment by paveljanda** [comment]
Thanks for the PR!

The `::onMach` event is needed though. (https://github.com/contributte/api-docu/blob/master/src/Starter.php#L95)
Could you please put it back?

**Comment by andrejsoucek** [comment]
Will do. So it is used in the other project to generate documentation?

File: src/ApiRoute.php [hunk 1]
```diff
@@ -11,18 +11,16 @@
...
 
 use Nette;
-use Nette\Application\IRouter;
 use Nette\Application\Request;
+use Nette\Routing\Router;
 use Nette\SmartObject;
 use Nette\Utils\Strings;
 
 /**
- * @method mixed onMatch(static, Nette\Application\Request $request)
```
Comment (paveljanda) [inline src/ApiRoute.php]: Could you please also keep this doc block? Thanks!

**Comment by paveljanda** [comment]
@andrejsoucek Exactly. This project works hand in hand with `api-docu` - documentation generating vendor. Thanks.

**Comment by f3l1x** [comment]
Please, take a look at our bare repo.

https://github.com/contributte/contributte/tree/master/bare

**Comment by andrejsoucek** [comment]
any progress?

**Comment by paveljanda** [comment]
@andrejsoucek Can you please resolve the conflicts? Thanks

**Comment by andrejsoucek** [comment]
should be ok now

**Comment by paveljanda** [comment]
Thanks!

**Pull request merged**